### ISSUE TITLE
Support Cake >= 0.26.0

### ISSUE THREAD

**Issue by AdmiringWorm** [cake-contrib/Cake.Recipe#207]
Update all addins to versions built against Cake 0.26.0

*Most of the list is the same as the previous one #159*

- [x] Cake.AppVeyor (https://github.com/cake-contrib/Cake.AppVeyor/issues/10)
- [x] Cake.Codecov (Supported since [0.4.0](https://github.com/cake-contrib/Cake.Codecov/releases/tag/0.4.0))
- [x] Cake.Coveralls (https://github.com/cake-contrib/Cake.Coveralls/issues/28)
- [x] Cake.Figlet (Supported since 1.1.0)
- [x] Cake.Git (Supported since [0.17.0](https://github.com/cake-contrib/Cake_Git/releases/tag/v0.17.0))
- [x] Cake.Gitter (https://github.com/cake-contrib/Cake.Gitter/issues/31)
- [x] Cake.Graph (https://github.com/cake-contrib/Cake.Graph/issues/11)
- [x] Cake.Incubator (Supported since [2.0.1](https://github.com/cake-contrib/Cake.Incubator/releases/tag/2.0.1))
- [x] Cake.Kudu (Supported since [0.7.0](https://github.com/cake-contrib/Cake.Kudu/releases/tag/v0.7.0))
- [x] Cake.MicrosoftTeams (Supported since [0.7.0](https://github.com/cake-contrib/Cake.MicrosoftTeams/releases/tag/v0.7.0))
- [x] Cake.ReSharperReports (https://github.com/cake-contrib/Cake.ReSharperReports/issues/33)
- [x] Cake.Slack (Supported since [0.11.0](https://github.com/cake-contrib/Cake.Slack/releases/tag/v0.11.0)
- [x] Cake.Transifex (Supported since [0.6.0](https://github.com/cake-contrib/Cake.Transifex/releases/tag/0.6.0))
- [x] Cake.Twitter (https://github.com/cake-contrib/Cake.Twitter/issues/27)
- [x] Cake.Wyam (https://github.com/Wyamio/Wyam/issues/649)

*Could be I missed someone, I'll append those to the list when I know which ones*

**Comment by pascalberger** [comment]
`Cake.CodeAnalysisReporting` is no longer used here. I therefore removed it from this list

**Comment by gep13** [comment]
This is what is currently being used:

```
#addin nuget:?package=Cake.AppVeyor&version=1.1.0.9
#addin nuget:?package=Cake.Codecov&version=0.3.0
#addin nuget:?package=Cake.Coveralls&version=0.7.0
#addin nuget:?package=Cake.Figlet&version=1.0.0
#addin nuget:?package=Cake.Git&version=0.16.1
#addin nuget:?package=Cake.Gitter&version=0.7.0
#addin nuget:?package=Cake.Graph&version=0.4.0
#addin nuget:?package=Cake.Incubator&version=1.7.2
#addin nuget:?package=Cake.Kudu&version=0.6.0
#addin nuget:?package=Cake.MicrosoftTeams&version=0.6.0
#addin nuget:?package=Cake.ReSharperReports&version=0.8.0
#addin nuget:?package=Cake.Slack&version=0.10.0
#addin nuget:?package=Cake.Transifex&Version=0.4.0
#addin nuget:?package=Cake.Twitter&version=0.6.0
#addin nuget:?package=Cake.Wyam&version=1.2.0
```

**Comment by gep13** [comment]
@AdmiringWorm think you got them all :+1:

**Comment by daveaglick** [comment]
Cake.Wyam is updated in source, will go out with the next release

**Comment by gep13** [comment]
Thanks to an amazing effort by @nmbro we only have two addin's remaining to have the necessary work merged into their repositoriesl

I have reached out to @redth to see if he would like me to take care of the necessary work, and I am hoping that @wozzo will be able to help out with getting the PR for Cake.Graph merged in.  If not, I can see about taking a look at that one as well.

Thanks again @nmbro amazing work!

**Comment by gep13** [comment]
@daveaglick do you have a timeline of when you are planning the next Wyam release?

**Comment by gep13** [comment]
@daveaglick not a problem, just wanted to touch base.  Hope you get it all worked out!

**Comment by wozzo** [comment]
Cake.Graph should be ready now too thanks to @nmbro

**Comment by gep13** [comment]
Cake.AppVeyor should be good to go with version 3.0.0

**Comment by AdmiringWorm** [comment]
awesome, now only a release for Cake.Wyam is missing

@daveaglick hint hint 😄

**Comment by daveaglick** [comment]
So close!

**Comment by daveaglick** [comment]
I've been blocked by MSBuild failing while performing code analysis in the docs recipe for days. Finally made some progress, turns out I just needed to add back some binding redirects. Should pave the way for a long-overdue release tomorrow or the next day.

**Comment by daveaglick** [comment]
Rolling out now

**Comment by gep13** [comment]
@daveaglick nice one!

**Comment by gep13** [comment]
Cake.Wyam should now be supported in version 1.4.0.  @nmbro can you update your PR again, and we can take this for a spin?

**Comment by nmbro** [comment]
@gep13 there you go

**Comment by JenasysDesign** [comment]
I did an interesting search of Github, for all the projects that use Cake.Recipe to see if I could find any know github repositories that would break if the issue and associated PR's were completed.

https://github.com/search?q=Cake.Recipe+filename%3Acake.build+in%3Afile&type=Issues

It does look like all the addin's in the original report are completed.

**Comment by gep13** [comment]
Taking this for a spin now...

**Comment by gep13** [comment]
Looking promising...
